apps/namehashlabs.org/components/2 - molecules/contact-us-form.tsx

```diff
@@ -272,27 +272,18 @@ export const ContactUsForm = ({ title }: ContactUsFormProps) => {
               >
                 Message
               </label>
-              <textarea
+              <TextArea
                 rows={4}
                 id="message"
                 disabled={isLoading}
                 onChange={handleInputChange}
                 defaultValue=""
                 name={FormFields.Message}
-                className={cc([
-                  "block w-full rounded-md px-3.5 py-2 text-gray-900 shadow-sm border border-gray-300 placeholder:text-gray-400 focus:border-gray-600 hover:border-gray-400 outline-none sm:text-sm sm:leading-6",
-                  {
-                    "border-red-300": validationErrors[FormFields.Message],
-                  },
-                ])}
+                error={validationErrors[FormFields.Message]}
               />
-
-              {validationErrors[FormFields.Message] && (
-                <span className="mt-2 text-sm font-normal text-red-600">
-                  {validationErrors[FormFields.Message]}
-                </span>
-              )}
             </div>
+            {/* Hidden input to capture the source URL of the form submission.
```

Comment:
Let's please keep the code in sync between both variations of this contact form for now. I've created a separate story for us to merge this duplicated logic into a new package in a separate PR :+1: